app/ide-desktop/lib/dashboard/src/components/AriaComponents/Form/Form.tsx

```diff
@@ -80,7 +80,7 @@ export const Form = React.forwardRef(function Form<
   React.useImperativeHandle(formRef, () => innerForm, [innerForm])
 
   const formMutation = reactQuery.useMutation({
-    mutationKey: ['FormSubmit', testId, id],
+    mutationKey: ['Form submition', `testId: ${testId}`, `id: ${id}`],
```

Comment:
`submission`
also not a fan of hardcoding parameter names using template literals - please use an object instead.

Reply:
> also not a fan of hardcoding parameter names using template literals - please use an object instead.

Yeah, me too, but I found this better to review by eyes compared to object literals:
![image](https://github.com/enso-org/enso/assets/61194245/10b98226-98fd-4026-9eb2-18b7fccc2121)

This mutation exists only for debug purposes - React Query dev tools will record the mutation, the result, and the variables(form fields).

In general, object literals are preferred

Reply:
i guess it would be nice to add a comment explaining why this specific log is like this (and to make it clear that we prefer object literals everywhere else)